Albums/Albums/AlbumsTableViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AlbumsTableViewController.swift
+//  Albums
+//
+//  Created by Enrique Gongora on 3/9/20.
+//  Copyright © 2020 Enrique Gongora. All rights reserved.
+//
+
+import UIKit
+
+class AlbumsTableViewController: UITableViewController {
+
+    var albumController: AlbumController?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        albumController?.getAlbums(completion: { (error) in
+            if let error = error {
+                NSLog("Error getting albums: \(error)")
+                return
+            }
+            self.tableView.reloadData()
```

Comment:
You will want to call this on using DisptachQueue to put it on the main thread since the getAlbums call is on a background thread.